### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by EvanHerman**
### Description
Update package dependencies.

### Types of changes
No new code, updated dependencies.

### Checklist:
- [x] My code is tested

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,6 +131,16 @@
 		"style-loader": "^0.19.1",
 		"webpack-fix-style-only-entries": "^0.4.0"
 	},
+	"resolutions": {
```
Comment (olafleur) [inline package.json:134]: Nice! I did not know that existed!

**Review by olafleur — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (EvanHerman) [reply package.json:134]: Sadly, I had to go this route because some nested dependencies were outdated.

Comment (jonathanbardo) [reply package.json:134]: @olafleur only in yarn :)

Comment (olafleur) [reply package.json:134]: @EvanHerman : yeah, not always an ideal situation, but it is sometimes a useful tool, like in this case :)

**Comment by AnthonyLedesma** [comment]
I always prefer to execute all E2E tests for an extra level of confidence that cascading issues do not exist within our software due to these dependency updates. I've re-branched this so we can watch tests here:

https://app.circleci.com/pipelines/github/godaddy-wordpress/coblocks?branch=fix%2Fdependencies-run-all-tests

**Review by AnthonyLedesma — APPROVED** [review]
LGTM. Tests are good.
image

https://dashboard.cypress.io/projects/sovnn2/runs/1394/overview

**Pull request merged**